### PR TITLE
Add pdpd fuzz test

### PR DESCRIPTION
**Pull request by asomsiko**
Adds fuzzing test for frontend API importing PDPD models

### CODE REVIEW

File: tests/fuzz/README.md [hunk 1]
```diff
@@ -18,14 +18,63 @@ To run fuzzing you will need [LLVM](https://apt.llvm.org/) components:
...
+
+1. Prepare fuzzing corpus
+
+Fuzzing engine needs a set of valid inputs to start fuzzing from. Those files
+are called a fuzzing corpus. Use tests/fuzz/scripts/init_corpus.py script to
+prepare fuzzing corpus.
+
+```bash
+tests/fuzz/scripts/init_corpus.py ./pdpd_layer_models/**/*.pdmodel --join pdiparams && \
+mkdir -p corpus && find ./pdpd_layer_models/ -name "*.fuzz" -exec cp \{\} ./import_pdpd-corpus \;
```
Comment (nosovmik) [inline tests/fuzz/README.md:60]: should it be 'mkdir -p import_pdpd_corpus'?

File: ngraph/frontend/paddlepaddle/src/op/pool2d.cpp [hunk 2]
```diff
@@ -66,7 +66,7 @@ namespace ngraph
                                         static_cast<uint64_t>(paddings[3])};
                         break;
                     default:
-                        throw std::runtime_error("Unsupported pooling paddings " + paddings.size());
+                        throw std::runtime_error("Unsupported pooling paddings " + std::to_string(paddings.size()));
```
Comment (nosovmik) [inline ngraph/frontend/paddlepaddle/src/op/pool2d.cpp:69]: CMake Error at /home/runner/work/openvino/openvino/cmake/developer_package/clang_format/clang_format_check.cmake:15 (message):
  [clang-format] Code style check failed for:
  /home/runner/work/openvino/openvino/ngraph/frontend/paddlepaddle/src/op/pool2d.cpp

File: tests/fuzz/README.md (hunk 1 above)
Comment (asomsiko) [reply tests/fuzz/README.md:60]: good catch!

File: ngraph/frontend/paddlepaddle/src/op/pool2d.cpp (hunk 2 above)
Comment (asomsiko) [reply ngraph/frontend/paddlepaddle/src/op/pool2d.cpp]: fixed

File: tests/fuzz/README.md (hunk 1 above)
Comment (asomsiko) [reply tests/fuzz/README.md]: fixed

**Pull request merged**